2021-05-27 23:23:40,849 INFO    MainThread:22202 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 23:23:40,849 INFO    MainThread:22202 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 23:23:40,850 INFO    MainThread:22202 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_232340-3dz17kfg/logs/debug.log
2021-05-27 23:23:40,850 INFO    MainThread:22202 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_232340-3dz17kfg/logs/debug-internal.log
2021-05-27 23:23:40,850 INFO    MainThread:22202 [wandb_init.py:init():369] calling init triggers
2021-05-27 23:23:40,850 INFO    MainThread:22202 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 1, 'batch_size': 16, 'init_lr': 0.002706761092128189}
2021-05-27 23:23:40,850 INFO    MainThread:22202 [wandb_init.py:init():418] starting backend
2021-05-27 23:23:40,850 INFO    MainThread:22202 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 23:23:40,859 INFO    MainThread:22202 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 23:23:40,864 INFO    MainThread:22202 [backend.py:ensure_launched():136] started backend process with pid: 22211
2021-05-27 23:23:40,865 INFO    MainThread:22202 [wandb_init.py:init():423] backend started and connected
2021-05-27 23:23:40,866 INFO    MainThread:22202 [wandb_init.py:init():465] updated telemetry
2021-05-27 23:23:40,867 INFO    MainThread:22202 [wandb_init.py:init():484] communicating current version
2021-05-27 23:23:41,617 INFO    MainThread:22202 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.31 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-05-27 23:23:41,617 INFO    MainThread:22202 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 23:23:41,847 INFO    MainThread:22202 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 23:23:41,890 INFO    MainThread:22202 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 23:23:41,891 INFO    MainThread:22202 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 23:23:41,891 INFO    MainThread:22202 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 23:23:41,903 INFO    MainThread:22202 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 23:23:41,903 INFO    MainThread:22202 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 23:26:51,514 INFO    MainThread:22202 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 1
2021-05-27 23:26:51,515 INFO    MainThread:22202 [wandb_run.py:_restore():1507] restore
2021-05-27 23:26:53,522 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:26:53,751 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 23:26:53,916 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3415
}

2021-05-27 23:26:54,341 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3417
  total_bytes: 3830
}

2021-05-27 23:26:54,443 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:54,544 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:54,645 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:54,746 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:54,848 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:54,949 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:55,050 INFO    MainThread:22202 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3830
  total_bytes: 3830
}

2021-05-27 23:26:56,146 INFO    MainThread:22202 [wandb_run.py:_show_files():1879] logging synced files
